### PR TITLE
Update Readme docs with the updated About icon in Instrument Studio

### PR DESCRIPTION
**Pull request by MounikaBattu17**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md). _(Required)_
- [ ]  Automatically post PR comments with images for G code changes? _(Recommended for small changes)_

### What does this Pull Request accomplish?
Updated Readme docs `Running the measurement.png` with the new official about icon in Instrument Studio.

### Why should this Pull Request be merged?
- Updated `Running the measurement.png` with the new About icon.
- Nit: Changed text alignment to Left Justify for description URL.

### What testing has been done?
None.

### CODE REVIEW

File: Source/Generator/Measurement Plug-In Template/Measurement/Get Service Descriptor.vi
Comment (MounikaBattu17) [inline Source/Generator/Measurement Plug-In Template/Measurement/Get Service Descriptor.vi:1]: Before:
image
After:
image

**Review by Avinash2Suresh — APPROVED** [review]

**Review by dixonjoel — APPROVED** [review]

File: Source/Generator/Measurement Plug-In Template/Measurement/Get Service Descriptor.vi
Comment (bkeryan) [reply Source/Generator/Measurement Plug-In Template/Measurement/Get Service Descriptor.vi:1]: URLs use forward slashes. This picture shows backslashes. Someone should check all of the URL strings in this component.

Comment (bkeryan) [reply Source/Generator/Measurement Plug-In Template/Measurement/Get Service Descriptor.vi:1]: Also, url dot com is owned by a domain squatter. Please use either example.com or an NI-controlled domain name.

Comment (MounikaBattu17) [reply Source/Generator/Measurement Plug-In Template/Measurement/Get Service Descriptor.vi:1]: Sure! Thanks for pointing this out. The change for this VI is only the text alignment of `description URL` control. The picture uses dummy URLs to showcase text alignment. The URL field in the control is left empty for users to input their URLs.
image

Comment (bkeryan) [reply Source/Generator/Measurement Plug-In Template/Measurement/Get Service Descriptor.vi:1]: Oh, I see. If the URL isn't actually saved in the VI then it's fine.

**Pull request merged**